testsuite/gnat2goto/tests/abs/test.out

```diff
@@ -1,7 +1,19 @@
+Standard_Error from gnat2goto absolute:
+Warning: Multi-language analysis unsupported.
```

Comment:
for this it might be worth to add something like

```
prove(ignore_lines=[
  'Warning: Multi-language analysis unsupported.'
  ])
```

which would drop all lines matching some pattern from the output

Reply:
See my comment about ignoring pragma Import when the calling convention is "Intrinsic" in PR#234

Reply:
Another option is to use the standard warning mechanism of the gnat front end to generate warning messages and then we could use the pragma Warnings to control reporting.